common/app/views/fragments/mostPopularPlaceholder.scala.html

```diff
@@ -13,9 +15,22 @@ <h2 class="fc-container__header__title">
                     <a href="@LinkTo(url)" data-link-name="Most viewed @section">most viewed</a>
                 </h2>
             </div>
-            <div class="fc-container__body fc-container--rolled-up-hide js-popular-trails">
-
+            <div class="fc-container__body fc-container--rolled-up-hide popular-trails">
+                <div class="popular-trails__content js-popular-trails">
+                </div>
+                <div class="popular-trails__mpu">
+                    <div class="fc-slice__popular-mpu js-fc-slice-mpu-candidate fc-slice__item--mpu-candidate">
```

Comment:
We close slots when `dfpAdvertising` is false. Does that mean the enclosing element has dimensions?